content/sensu-go/5.10/reference/datastore.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Datastore"
+description: "Sensu stores the most recent event for each entity and check pair using an embedded etcd (default) or an external etcd instance. Sensu also supports using an external PostgreSQL instance for event storage in place of etcd (licensed-tier only). Read the reference to configure enterprise-scale event storage using PostgreSQL."
+weight: 10
+version: "5.10"
+product: "Sensu Go"
+menu:
+  sensu-go-5.10:
+    parent: reference
+---
+
+- [Event storage](#event-storage)
+- [Scaling event storage](#scaling-event-storage) (licensed-tier only)
+  - [Requirements](#requirements)
+  - [Configuration](#configuration)
+  - [Specification](#specification)
+
+## Event storage
+
+Sensu stores the most recent event for each entity and check pair using an embedded etcd (default) or an [external etcd][8] instance.
+You can access event data using the [dashboard][9] events page, [`sensuctl event` commands][10], and the [events API][11].
+To store event data over time, we recommend integrating Sensu with a time series database, like [InfluxDB][12].
+
+## Scaling event storage
+
+**LICENSED TIER**: Unlock enterprise-scale event storage in Sensu Go with a Sensu license. To activate your license, see the [getting started guide][13].
+
+Sensu supports using an external PostgreSQL instance for event storage in place of etcd.
+PostgreSQL can handle significantly higher volumes of Sensu events, letting you scale Sensu beyond etcd's 8GB limit.
+
+Configured with a PostgreSQL event store, Sensu connects to PostgreSQL to store and retrieve event data in place of etcd, while etcd continues to store Sensu configuration data.
```

Comment:
```suggestion
Configured with a PostgreSQL event store, Sensu connects to PostgreSQL to store and retrieve event data in place of etcd, while etcd continues to store Sensu entity and configuration data.
```

Reply:
Thanks, @cwjohnston! Can you elaborate on this suggestion?

Reply:
I'm thinking about this as a way to suggest or clarify that etcd is still being used as a registry for things which aren't events and also aren't configuration, strictly speaking.